WordPress/Classes/ViewRelated/Menus/Views/MenuItemEditingFooterView.m

```diff
@@ -0,0 +1,90 @@
+#import "MenuItemEditingFooterView.h"
+#import "WPStyleGuide.h"
+#import "WPFontManager.h"
+
+@import Gridicons;
+
+@interface MenuItemEditingFooterView ()
+
+@property (nonatomic, strong) IBOutlet UIButton *cancelButton;
+@property (nonatomic, strong) IBOutlet UIButton *trashButton;
+@property (nonatomic, strong) IBOutlet UIButton *saveButton;
+
+@end
+
+@implementation MenuItemEditingFooterView
+
+- (void)awakeFromNib
+{
+    [super awakeFromNib];
+
+    [self setupCancelButton];
+    [self setupTrashButton];
+    [self setupSaveButton];
+}
+
+- (void)setupCancelButton
+{
+    UIButton *button = self.cancelButton;
+    button.titleLabel.font = [WPFontManager systemRegularFontOfSize:18.0];
+    [button setTitleColor:[WPStyleGuide darkGrey] forState:UIControlStateNormal];
+    [button setTitleColor:[WPStyleGuide greyDarken20] forState:UIControlStateHighlighted];
+    [button setTitle:NSLocalizedString(@"Cancel", @"Menus: Cancel button title for canceling an edited menu item.") forState:UIControlStateNormal];
+    [button addTarget:self action:@selector(cancelButtonPressed) forControlEvents:UIControlEventTouchUpInside];
+    button.backgroundColor = [UIColor clearColor];
```

Comment:
You might consider moving some of the button configuration into the xib rather than define it here.  IBAction, backgroundColor, font/size would be good candidates. 
Same for the other buttons. 
Just a thought.